admin_ui_support/modules/jsonapi_tables/jsonapi_tables.info.yml

```diff
@@ -0,0 +1,7 @@
+name: 'JSONAPI Table'
```

Comment:
Should we mark this as experimental 🤔

Reply:
I would assume this whole repo is experimental right now so don't think there is need to mark this.